modules/nf-core/propr/propd/main.nf

```diff
@@ -10,17 +10,21 @@ process PROPR_PROPD {
     input:
     tuple val(meta), path(count)
     tuple val(meta2), path(samplesheet)
+    val GSEA
```

Comment:
I think in this case lower case is preferred.
Also, only the very needed input should stay here. This kind of optional input should stay in ext.args.
If you are not used to ext.args, check https://github.com/nf-core/differentialabundance/blob/master/conf/modules.config
You can also check how I call ext.args inside the test configs.

Reply:
Fair enough, I will remove the optional boolean from the propd module and just leave it in the ext.args with False as default.